### ISSUE TITLE
Multi-Resolution Camera Frame will remain hidden if in the wrong collection

### ISSUE THREAD

**Issue by Gatada** [Gatada/MultiResolutionCameras#3]
## SYMPTOM
Rendering frame does not appear in the 3D Viewport when the Active Camera has a custom rendering resolution, despite the mesh icon being correctly parented to the camera.

## ISSUE
I believe the issue being that the Frame Mesh was initially created in a collection that is currently hidden in viewport, while the Active Camera is in another visible collection.

## SUGGESTED SOLUTION
Since a mesh can be a member of multiple collections, maybe ensuring that the Frame Mesh is only found in the collection containing the Active Camera will solve the issue?

**In short:** Move Render Frame to the collection used by the active camera.

**Comment by Gatada** [comment]
Seems this is worth trying:

> You just have to [.unlink(object)](https://docs.blender.org/api/current/bpy.types.CollectionObjects.html#bpy.types.CollectionObjects.unlink) each object from all [linked collections](https://blender.stackexchange.com/a/134789/31447) and then [.link(object)](https://docs.blender.org/api/current/bpy.types.CollectionObjects.html#bpy.types.CollectionObjects.link) each object to your target collection.

[Source](https://blender.stackexchange.com/questions/175146/how-to-move-multiple-objects-into-another-collection)

**Comment by Gatada** [comment]
Testing suggests that 3.2.10 fixes the issue.
